### PR TITLE
Added template to the bottom

### PR DESCRIPTION
**Pull request by mezotv**
This is to make it easier for new people to add their projects with a small template.

### CODE REVIEW

**Review by ykdojo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -256,6 +256,13 @@ https://github.com/commclassroom/roadmaps
 
 ---
 
+## Your Project
+> Make sure to include a small but informative description on what your project is about and how we can help.
```
Comment (ykdojo) [inline README.md:260]:
```suggestion
> Make sure to include a small but informative description on what this project is about.
```

File: README.md [hunk 2]
```diff
@@ -256,6 +256,13 @@ https://github.com/commclassroom/roadmaps
...
 
+Built on: Libraries and languages your project is build with.
```
Comment (ykdojo) [inline README.md:262]:
```suggestion
Built on: Libraries and languages this project is built with.
```

File: README.md [hunk 3]
```diff
@@ -256,6 +256,13 @@ https://github.com/commclassroom/roadmaps
...
+
+`Your GitHub repository url`
```
Comment (ykdojo) [inline README.md:264]:
```suggestion
`GitHub repository url`
```

**Review by ykdojo — APPROVED** [review]

**Comment by mezotv** [comment]
Thanks for merging.

**Pull request merged**